packages/storybook-builder/package.json

```diff
@@ -49,10 +49,11 @@
   "dependencies": {
     "@rollup/plugin-node-resolve": "^15.1.0",
     "@rollup/pluginutils": "^5.0.2",
-    "@storybook/core-common": "^7.0.0",
+    "@storybook/channels": "^8.3.3",
```

Comment:
for consistency across all Storybook packages it's important to have same version
as a library we didn't force any specific one and tried to be compatible with every
updating all storybook packages and keeping them on the same version is hard enough task, so the library shouldn't make it even harder
this is different here
any specific reason?

Reply:
not sure what do you mean

I had to add `@storybook/core` because channel transports have been moved there from `@storybooks/channels`

Reply:
I think what he means is that here you're dependending on `^8.3.3` while below you're depending on `^8.0.0`, and that they should be the same across the board

Reply:
ah okay, I will upgrade it to 8.3 everywhere, if I haven't already